pycaret/time_series/forecasting/oop.py

```diff
@@ -3089,12 +3101,17 @@ def _plot_model(
         if display_format == "streamlit":
             _check_soft_dependencies("streamlit", extra=None, severity="error")
             import streamlit as st
+        elif display_format == "plotly-widget":
+            _check_soft_dependencies("IPython", extra=None, severity="error")
```

Comment:
I think IPython is not a soft dependency. It is installed in the default install.

Reply:
Good catch, you're right!